tutorials/_posts/2020-10-18-transfer-learning.md

```diff
@@ -78,8 +78,9 @@ loss(x,y) = Flux.Losses.logitcrossentropy(model(x), y)
 Now to train. As discussed earlier, we don’t need to pass all the parameters to our training loop. Only the ones we need to fine-tune. Note that we could have picked and chosen the layers we want to train individually as well, but this is sufficient for our use as of now.
 
 ```julia
-ps = Flux.params(model[2:end])  # ignore the already trained layers of the ResNet
+ps = Flux.params(model)
 ```
+Normally, you would only re-train the dense layers via `model[2:end]` but in this case, the pre-trained models from Metalhead.jl are not currently available. This will change in the future once the pre-trained models are once again available.
```

Comment:
Can this use some kind of warning or info formatting? e.g. I know Documenter.jl has `!!!` and co.

Reply:
We could embed some Javascript but not sure that is the best option. I will have a re-built version of the Flux site using Franklin by end of year so perhaps it is okay to just use plain text for now?

Reply:
Oh nice, I was wondering about that. How about just slapping a bold **Note:** at the start for now?

Reply:
```suggestion
**Note**: Normally, you would only re-train the dense layers via `model[2:end]` but in this case, the pre-trained models from Metalhead.jl are not currently available. This will change in the future once the pre-trained models are once again available.
```